backend/controller/controller.go

```diff
@@ -1471,14 +1472,19 @@ func (s *Service) reserveRunner(ctx context.Context, reconcile model.Deployment)
 		"languages": []string{reconcile.Language},
 	})
 	if err != nil {
-		return clients{}, fmt.Errorf("failed to claim runners for %s: %w", reconcile.Key, err)
+		return clients{}, fmt.Errorf("failed to claim runners for %s %s: %w", reconcile.Key, reconcile.Module, err)
 	}
 
 	err = dal.WithReservation(reservationCtx, claim, func() error {
 		client = s.clientsForEndpoint(claim.Runner().Endpoint)
 		_, err = client.runner.Reserve(reservationCtx, connect.NewRequest(&ftlv1.ReserveRequest{DeploymentKey: reconcile.Key.String()}))
-		return err
+		if err != nil {
+			return fmt.Errorf("failed to request to reserve a runner for %s %s: %w", reconcile.Key, reconcile.Module, err)
```

Comment:
I think we can get rid of the first `to` to make it read a bit better

Reply:
Let's add the endpoint here though as thats the missing piece of info we need to know if we think it should be able to connect ok